app/src/test/java/com/monstarlab/features/auth/domain/AuthRepositoryTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.monstarlab.features.auth.domain
+
+import com.monstarlab.features.auth.AuthTokenMapper
+import com.monstarlab.features.auth.data.api.AuthApi
+import com.monstarlab.features.auth.data.api.dtos.TokenResponseDTO
+import com.monstarlab.features.auth.domain.models.AuthToken
+import io.mockk.*
+import junit.framework.TestCase.assertEquals
+import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Created by aslam on 08,March,2023
+ */
+internal class AuthRepositoryTest {
+
+    private lateinit var mockAuthApi: AuthApi
+    private lateinit var mockAuthTokenMapper: AuthTokenMapper
+    private lateinit var authRepository: AuthRepository
+
+    @Before
+    fun setUp() {
+        mockAuthApi = mockk()
+        mockAuthTokenMapper = mockk()
+        authRepository = AuthRepository(mockAuthApi, mockAuthTokenMapper)
+    }
+
+    @After
+    fun tearDown() {
+        clearAllMocks()
+    }
+
+    @Test
+    fun testLoginReturnsExpectedAuthToken() = runBlocking {
```

Comment:
Please use `runTest` instead of `runBlocking`.  Its available in the coroutines-test artifact